src/stepFunctions/commands/visualizeStateMachine/aslVisualizationManager.ts

```diff
@@ -35,7 +35,8 @@ export class AslVisualizationManager {
          * Ensure tests are written for this use case as well.
          */
 
-        if (!activeTextEditor) {
+        // Output channel is considered a text editor by VSCode (which we don't want)
+        if (!activeTextEditor || activeTextEditor.document.fileName.includes('extension-output')) {
```

Comment:
Seems like it should be in a different PR

Reply:
It could be a separate PR but it's needed for the integration tests to work. When the Go extension installs `gopls` it prints to the output channel and causes the test to break. The test checks for no active text editors, but the output channel counts as one.

Reply:
Can you add that to the comment that that is the reason?